vncviewer/Viewport.cxx

```diff
@@ -632,9 +632,19 @@ void Viewport::flushPendingClipboard()
 }
 
 
+void Viewport::handleMenuClosed(void *data)
+{
+  Viewport *self = (Viewport *)data;
+  assert(self);
+  self->menuOpened = false;
+}
+
+
 void Viewport::handlePointerEvent(const core::Point& pos,
                                   uint16_t buttonMask)
 {
+  if (menuOpened)
+    return;
```

Comment:
This is very non-obvious, so a comment would be nice, explaining that this is a workaround and for what